daemon/lokinet.cpp

```diff
@@ -84,9 +86,6 @@ install_win32_daemon()
     llarp::LogError("Cannot install service ", GetLastError());
     return;
   }
-  // just put the flag here. we eat it later on and specify the
-  // config path in the daemon entry point
-  StringCchCat(szPath.data(), 1024, " --win32-daemon");
```

Comment:
This seems wrong.  If removed here it seems there is nowhere that sets the service up with the `--win32-daemon` flag, but main() still relies on it?

Reply:
yea. i also dont know.

Reply:
Why did you remove it?

Reply:
it was using a function called `StringCchCat` from some windows header that does god knows what. i forget to figure out wtf it did and emulate the desired behavior.